.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,21 +1,30 @@
-> Read our [Contribution guide](https://github.com/eclipse/che-docs/blob/master/CONTRIBUTING.adoc) before submitting a PR.
 
-### What does this PR do?
+<!-- 
+Please use one of the following prefixes for the title:
+docs: Documentation not including procedures. Engineering review is mandatory.
+procedures: Documentation including procedures. Testing procedures is mandatory. Engineering and QE review is mandatory (Engineering can review on behalf of QE). 
+chore: Routine, release, tooling, version upgrades.
+fix: Fix build, language, links, or metadata.
+-->
 
+<!-- Read our [Contribution guide](https://github.com/eclipse/che-docs/blob/master/CONTRIBUTING.adoc) before submitting a PR. -->
 
-### What issues does this PR fix or reference?
+## What does this PR change?
 
+## What issues does this PR fix or reference?
 
-### Specify the version of the product this PR applies to.
+## Specify the version of the product this PR applies to
 
+## Pull Request checklist
 
-### PR Checklist
+The author and the reviewers validate the content of this Pull Request with the following checklist, in addition to the [automated tests](code_review_checklist.adoc).
 
-As the author of this Pull Request I made sure that:
-
-- [ ] `vale` has been run successfully against the PR branch
-- [ ] Link checker has been run successfully against the PR branch
-- [ ] Documentation describes a scenario that is already covered by QE tests, otherwise an issue has been created and acknowledged by Che QE team
-- [ ] Changed article references are updated where they are used (or a redirect has been configured on the docs side):
+- Any procedure:
+  - [ ] Successfully tested.
+- Any page or link rename:
+  - [ ] The page contains a redirection for the previous URL.
+  - Propagate the URL change in:
     - [ ] Dashboard [default branding data](https://github.com/eclipse-che/che-dashboard/blob/main/src/services/bootstrap/branding.constant.ts)
     - [ ] Chectl [constants.ts](https://github.com/che-incubator/chectl/blob/master/src/constants.ts)
+- [ ] Builds on [Eclipse Che hosted by Red Hat](https://workspaces.openshift.com).
```

Comment:
Do we really need this?
1. Not all people do the development online
2. For external contributors it could be an additional obstacle.

Reply:
All the tasks don't need to be completed by the contributor. At least one of the reviewer should take care of it. Should we make it more clear?

I believe it's important to test there, as it is a good way to make sure dogfooding is never lest aside.

Reply:
Agree that documentation testing is crucial. If this requirement helps us with getting more PRs actually tested, let's keep it in.